scripts/deb/post-install.sh

```diff
@@ -51,6 +51,16 @@ test -d $LOG_DIR || mkdir -p $LOG_DIR
 chown -R -L telegraf:telegraf $LOG_DIR
 chmod 755 $LOG_DIR
 
+STATE_DIR=/var/lib/telegraf
+test -d $STATE_DIR || mkdir -p $STATE_DIR
+chown -R -L root:telegraf $STATE_DIR
+chmod 770 $STATE_DIR
+
+STATE_FILE=$STATE_DIR/statefile
+test -f $STATE_FILE || touch $STATE_FILE
+chown -R -L root:telegraf $STATE_FILE
+chmod 660 $STATE_FILE
```

Comment:
Similarly here, I wonder if this is better (note best practice of also quoting the shell variables (this is a nitpick since nothing here is user controllable; it just has better style):
```
STATE_FILE="$STATE_DIR/statefile"
test -f "$STATE_FILE" || {
    touch -h "$STATE_FILE"
    chown root:telegraf "$STATE_FILE"
    chmod 660 "$STATE_FILE"
}
```